website/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
+    "start": "LOCAL=true; docusaurus start",
```

Comment:
Not using this right now but I assume we will need some way to specify environment so development points to localhost

Reply:
you may be able to get flags from webpack.

https://webpack.js.org/guides/environment-variables/

try `!process.env.production`

Reply:
shouldn't this be `FUNCTIONLESS_LOCAL` to match above?